### DISCUSSION TITLE
Keeping user_data_dir between sessions

### DISCUSSION THREAD

**Discussion by vannman** [kaliiiiiiiiii/Selenium-Driverless#177] [Q&A] [answered]
When quitting a driver the user_data_dir is deleted. Is it possible to not delete in order to reuse it next time the driver is loaded?

I'm trying to maintain logged in sessions to sites and attempted to export and import cookies but it does not maintain the logged in session on some sites.

**Comment by ttlns** [comment, marked as answer]
https://github.com/kaliiiiiiiiii/Selenium-Driverless/issues/174

**Comment by vannman** [comment]
Beautiful!
